### PR TITLE
updated all dependent packages with the laravel 10 support version

### PR DESCRIPTION
**Pull request by swapnilsarwe**
This MR is to upgrade version of all packages with the latest laravel 10 support version.

### CODE REVIEW

**Comment by driesvints** [comment]
@swapnilsarwe were you still working on this?

**Comment by swapnilsarwe** [comment]
> @swapnilsarwe were you still working on this?

Oops lost track of this PR.
Will check what is the respective packages status & update

**Comment by swapnilsarwe** [comment]
> @swapnilsarwe were you still working on this?

Oops lost track of this PR.
Will check what is the status of the  respective packages & update

**Comment by swapnilsarwe** [comment]
just one package left 
https://github.com/nerdroid23/blade-icomoon/pull/4

PR already exists, i hope @nerdroid23 gets time to merge it soon.

**Comment by swapnilsarwe** [comment]
@driesvints - the last package has merged the PR with laravel 10 upgrade changes. but unfortunately maintainer missed out on tagging it. Hence we wont be able to upgrade to the latest version yet.

Have been following up, with the maintainer @nerdroid23 - i hope he does it sooner.

**Comment by swapnilsarwe** [comment]
@driesvints - we got all blade-icon packages updated with the laravel 10 support

feel free to review and merge this PR as per your convenience.

File: composer.json [hunk 1]
```diff
@@ -8,16 +8,16 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "~8.2.0",
+        "php": "^8.2",
```
Comment (driesvints) [inline composer.json:11]: Can you please undo this change? The site runs on 8.2 so that should be enforced.

**Review by driesvints — CHANGES_REQUESTED** [review]
Thanks @swapnilsarwe! Just one minor change.

File: composer.json (hunk 1 above)
Comment (swapnilsarwe) [reply composer.json]: done 👍🏽

**Comment by driesvints** [comment]
Thank you @swapnilsarwe!
